src/components/KToaster/ToastManager.ts

```diff
@@ -34,15 +34,15 @@ export default class ToastManager {
     this.toaster = createVNode(KToaster, {
       toasterState: this.toasts.value,
       zIndex: options?.zIndex ? options.zIndex : defaultZIndex,
-      onClose: (key: any) => this.close(key),
+      onClose: (key: string) => this.close(key),
```

Comment:
question: the interface is changing here; is that going to create a downstream issue?

Reply:
I believe the Konnect preview PR is good https://github.com/Kong/konnect-ui-apps/pull/6728 (but there are conflicts now).